lib/trento/user_identities.ex

```diff
@@ -53,6 +51,20 @@ defmodule Trento.UserIdentities do
     end
   end
 
+  defp maybe_create_user(nil, user_identity_params, user_params, user_id_params) do
```

Comment:
Most probably this function needs to go before `maybe_assign_global_abilities`, right?

Reply:
done! thx